charts/newrelic-pixie/templates/deployment.yaml

```
@@ -66,6 +66,12 @@ spec:
         - name: PIXIE_ENDPOINT
           value: "staging.withpixie.dev:443"
         {{- end }}
+        {{- if .Values.proxy }}
+        - name: HTTP_PROXY
+          value: "{{ .Values.proxy }}"
+        - name: HTTPS_PROXY
+          value: "{{ .Values.proxy }}"
+        {{- end }}
```

I would sugges to use `quote` since it will escape inner quotes rather than producing potentially incorrect yaml.

```suggestion
        {{- if .Values.proxy }}
        - name: HTTP_PROXY
          value: {{ .Values.proxy | quote }}
        - name: HTTPS_PROXY
          value: {{ .Values.proxy | quote }}
        {{- end }}
```